gtfs-netex-test/epip_db_to_db.py

```diff
@@ -41,9 +41,9 @@ def main(source_database_file: str, target_database_file: str):
     with Database(target_database_file, read_only=False) as target_db:
         setup_database(target_db, classes, True)
         # attach_source(con, source_database_file) does not work persistently, requires an attach at every connection
+        copy_table(source_database_file, target_db, [Notice, ScheduledStopPoint], clean=True)
```

Comment:
Don't want to go in this direction.

Reply:
but it worked :-)

Reply:
Yes, it does open the database, but it is read_only. But it does not make any sense that the _simple example_ (mail) does run, and the code below does not.